### DISCUSSION TITLE
Running cava with ghostty -e

### DISCUSSION THREAD

**Discussion by lokesh-krishna** [ghostty-org/ghostty#3767] [Q&A] [answered]
Running cava by launching an instance of ghostty works just fine but whenever I try to launch cava with `ghostty -e cava` or `ghostty --command=cava`, it just exits as soon as it launches.

Log: [ghostty](https://paste.low-key.me/low-key/3ef2b8e76266450a89970ba082d2940d)

This is the only output on the ghostty instance that's launched before it crashes: `free(): invalid next size (fast)`

**Comment by Anomalocaridid** [comment]
I am also experiencing this problem. For some reason, this also happens when running `ghostty -e sh -c cava` or `ghostty -e bash -c cava`, but not `ghostty -e fish -c cava`.

**Comment by artrixdotdev** [comment]
+1 this is happening for me too

**Comment by 00-kat** [comment, marked as answer]
Try `ghostty -e /path/to/cava`.  If you don't know what that path is, run `which cava`.
